packages/block/test/eip1559block.spec.ts

```diff
@@ -22,10 +22,14 @@ const genesis = Block.fromBlockData({})
 
 // Small hack to hack in the activation block number
 // (Otherwise there would be need for a custom chain only for testing purposes)
-common.getEIPActivationBlockNumber = function (eip: number) {
-  if (eip == 1559) {
+common.hardforkBlockBN = function (hardfork: string | undefined) {
+  if (hardfork === 'london') {
     return new BN(1)
+  } else if (hardfork === 'dao') {
+    // Avoid DAO HF side-effects
+    return new BN(99)
```

Comment:
What are these side effects?

Reply:
Tests were breaking due to the DAO HF being triggered on some test block(s) where block validation would then fail.